openhub/oh_token.py

```diff
@@ -0,0 +1,9 @@
+import logging
+
+from gci.config import get_api_key
+
+try:
+    OH_TOKEN = get_api_key('OH')
+except Exception as ex:
+    OH_TOKEN = None
+    logging.debug('OH_TOKEN can not be obtained: \n{}'.format(str(ex)))
```

Comment:
gah; why is this 'debug' level?  this is the error which causes the app to not work.